apps/mull-api/axiosConfig.ts

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+const axiosInstance = axios.create({
+  baseURL: `https://maps.googleapis.com/maps/api/place/autocomplete/json?input=845_Rue_Sherbrooke&key=${process.env.GOOGLE_KEY}`,
```

Comment:
Let me know if you need more clarity, we can hop on a quick call

Reply:
Yeah, initially this was done with `axios.get()` but the reason why theres an axios instance being created is because of the test that intercepts this call using ```jest.mock('../../../../axiosConfig', ...``` . We weren't able to intercept with the api call without an axios instance which is why the `axiosConfig.ts` was created

Reply:
I see, maybe we should only use it in tests for now until we figure out how to mock it. And maybe you could add documentation explaining the purpose of this file for future readers and under what conditions it could be removed

Reply:
Documentation added